src/runtime/virtcontainers/pkg/types/pcipath.go

```diff
@@ -0,0 +1,98 @@
+// Copyright Red Hat.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package types
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+const (
+	pciSlotBits = 5
+	maxPciSlot  = (1 << pciSlotBits) - 1
+)
+
+// A PciSlot describes where a PCI device sits on a single bus
+//
+// This encapsulates the PCI slot number a.k.a device number, which is
+// limited to a 5 bit value [0x00..0x1f] by the PCI specification
+//
+// XXX In order to support multifunction device's we'll need to extend
```

Comment:
This is a TODO? Should we open/site an issue so we don't lose track?

Reply:
Well, we've done without multifunction support here so far, so I don't see that it's a pressing concern.